cnd/migrations/2020-05-06-105000_created-swap-tables/up.sql

```diff
@@ -0,0 +1,64 @@
+-- Your SQL goes here
+
+CREATE TABLE swaps
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    local_swap_id UNIQUE	NOT NULL,
+    role			NOT NULL,
+    counterparty_peer_id	NOT NULL
+);
+
+CREATE TABLE secret_hashes
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    swap_id INTEGER		NOT NULL,
+    secret_hash		NOT NULL,
+    FOREIGN KEY (swap_id)	REFERENCES swaps(id)
+);
+
+CREATE TABLE shared_swap_ids
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    swap_id INTEGER		NOT NULL,
+    shared_swap_id		NOT NULL,
+    FOREIGN KEY (swap_id)	REFERENCES swaps(id)
+);
+
+CREATE TABLE address_hints
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    swap_id INTEGER		NOT NULL,
+    address_hint		NOT NULL,
+    FOREIGN KEY(swap_id)	REFERENCES swaps(id)
+);
+
+CREATE TABLE herc20_swap_details
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    swap_id INTEGER		NOT NULL,
+    amount			NOT NULL,
+    chain_id			NOT NULL,
+    expiry			NOT NULL,
+    hash_function		NOT NULL,
+    token_contract		NOT NULL,
+    redeem_identity,
+    refund_identity,
+    ledger			NOT NULL CHECK (ledger IN ('alpha', 'beta')),
```

Comment:
Should we rename this to `side` to avoid confusion with what we usually call ledger?

```suggestion
    side			NOT NULL CHECK (side IN ('alpha', 'beta')),
```